cmd/juju/status/status.go

```diff
@@ -389,6 +389,10 @@ func (c *statusCommand) runStatus(ctx *cmd.Context) error {
 		if err != nil {
 			return err
 		}
+		// A change was made in cmd/v3.0.2 output.go that broke the consistency in output for the
+		// default formatter by removing the newline at the end.
+		// https://github.com/juju/cmd/commit/be22fa661a798055c801f1511aee226db249ef95
+		fmt.Println()
```

Comment:
I'm not entirely certain what the difference would be, but I suspect you should probably use ctx.Infof instead of an fmt method.

`ctx.Infof("\n")` would certainly do this. You could even just add the `\n` to the line below if you feel like it

Reply:
Oh yes. I know for sure ctx.Infof and its variants writes to the logger if ctx.Quiet is set. But a single `\n` delimiter won't hurt.